src/databricks/labs/lsql/dashboards.py

```diff
@@ -218,7 +229,9 @@ def _parse_header(self, header: str) -> dict:
 
     def split(self) -> tuple[str, str]:
         trimmed_content = self._content.strip()
-        return trimmed_content, ""
+        # Current filter implementation only uses the filter header; the body is not implemented
+        # We return a non-empty body to signal that we know the body is not implemented and not actually empty
+        return trimmed_content, "NOT IMPLEMENTED"
```

Comment:
Hmm. I think I'd prefer `None` here as such a signal because the linter will pick that up?

The way I see it, either:
 - the caller has to handle it, in which case `""` was problematic and `"NOT IMPLEMENTED"` improves the situation but still allows the caller to ignore it;
 - the caller is allowed to ignore it, because there's a meaningful default to use. To me `""` looks more like that kind of value, and `"NOT IMPLEMENTED"` does not.